src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -293,9 +272,16 @@ export const TradeInput = memo(() => {
     tradeQuoteStep?.source,
   ])
 
+  const isRefetching = useMemo(() => {
+    if (activeSwapperName) {
+      return isTradeQuoteApiQueryPending[activeSwapperName] ?? false
+    }
+    return false
+  }, [activeSwapperName, isTradeQuoteApiQueryPending])
```

Comment:
Nitpick, could check for explicit `true` value instead
```suggestion
  const isRefetching = useMemo(() => Boolean(activeSwapperName && isTradeQuoteApiQueryPending[activeSwapperName] === true), [activeSwapperName, isTradeQuoteApiQueryPending])
```

Reply:
way cleaner b0288f0867ece87756a1559a624fd6f081c65c66